Docs/sphinx_documentation/source/TimeIntegration_Chapter.rst

```diff
@@ -0,0 +1,101 @@
+
+.. _sec:basics:timeintegration:
+
+Time Integration
+================
+
+AMReX provides a basic explicit time integrator capable of Forward Euler or
+both predefined and custom Runge-Kutta schemes designed to advance data on a
+particular AMR level by a timestep. This integrator is designed to be flexible,
+requiring the user to supply a right-hand side function taking a ``MultiFab``
+of state data and filling a ``MultiFab`` of the corresponding right hand side
+data. The user simply needs to supply a C++ lambda function to implement
+whatever right hand side operations they need.
+
+A Simple Time Integrator Setup
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is best shown with some sample code that sets up a time integrator and
+asks it to step forwards by some interval ``dt``. The user needs to supply at
+minimum, the right-hand side function using the ``TimeIntegrator::set_rhs()``
+function. A user can also supply a post update function which is called on
+state data immediately before evaluating the right-hand side. This post update
+function is a good opportunity to fill boundary conditions for Runge-Kutta
+stage solution data so that ghost cells are filled when the right hand side
+function is called on that solution data.
```

Comment:
```suggestion
This is best shown with some sample code that sets up a time integrator and
asks it to step forwards by some interval ``dt``. The user needs to supply at
minimum, the right-hand side function using the ``TimeIntegrator::set_rhs()``
function. By using the ``TimeIntegrator::set_post_update()`` function, a user
can also supply a post update function which is called on state data immediately
before evaluating the right-hand side. This post update function is a good
opportunity to fill boundary conditions for Runge-Kutta stage solution data so that
ghost cells are filled when the right hand side function is called on that solution data.
```